lib/rich-text-codemirror-adapter.js

```diff
@@ -176,7 +176,10 @@ firepad.RichTextCodeMirrorAdapter = (function () {
   };
 
   RichTextCodeMirrorAdapter.prototype.onCursorActivity = function () {
-    this.trigger('cursorActivity');
+    var self = this;
```

Comment:
Feel like adding a comment on why this is necessary? Otherwise I'll probably forget and remove it. :-)